include/node_base.h

```diff
@@ -17,6 +17,14 @@
 
 namespace mpm {
 
+//! Particle phases
+enum NodePhase : unsigned int {
+  nSolid = 0,
```

Comment:
remove the `n` breaks our convention

Reply:
I tried to use the same "Solid" with the ParticlePhase, but it will conflict with the  ParticlePhase declaration in compiling.

Reply:
Let's do `NSolid`